source/Ship.cpp

```diff
@@ -735,6 +736,8 @@ string Ship::FlightCheck() const
 		return "limited turn?";
 	if(energy - .8 * solar < .2 * (turnEnergy + thrustEnergy))
 		return "solar power?";
+	if(fuel <= 0. && !canBeCarried)
```

Comment:
And if the ship requires 0 fuel to jump? You currently would still tell the player they are stuck.